### PR TITLE
JestJS BrowserStack documentation

### PR DESCRIPTION
**Pull request by NightWing1998**

### CODE REVIEW

File: src/single.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+const { Builder, By, Key, until } = require("selenium-webdriver");
+const https = require("https");
+
+describe("BStack demo test", () => {
+	let driver,
+		statusFail = true;
+
+	const username = process.env.BROWSERSTACK_USERNAME || "",
+		accessKey = process.env.BROWSERSTACK_ACCESS_KEY || "";
```
Comment (Archish27) [inline src/single.test.js:9]: Declare separate varaibles

**Review by Archish27 — CHANGES_REQUESTED** [review]
Please format the document. Also, add 2 spaces indentation in all files.

**Review by Archish27 — APPROVED** [review]
Can you run formatter on this project. looks like filr

**Comment by Archish27** [comment]
Merge conflicts

**Review by Archish27 — CHANGES_REQUESTED** [review]
merge conflicts

**Review by Archish27 — APPROVED** [review]

File: src/parallel.test.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+        expect(products).toHaveLength(25);
+
+        await driver.findElement(By.css("input[value='Apple'] + span")).click();
+
+        await driver
+          .findElement(By.css("input[value='Samsung'] + span"))
+          .click();
+
+        await driver.sleep(1500);
```
Comment (asambstack) [inline src/parallel.test.js:106]: Why not use something like
`await driver.wait(until.elementLocated());`

File: conf/single.conf.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+module.exports = {
+  os: "Windows",
+  os_version: "11",
+  browserName: "Chrome",
+  browser_version: "latest",
+  project: "BStack Demo",
+  build: process.env.BROWSERSTACK_BUILD_NAME || "jest-browserstack",
+  name: "single test",
+  "browserstack.local": "false",
+  "browserstack.selenium_version": "3.6.0",
```
Comment (asambstack) [inline conf/single.conf.js:15]: Is this required?

**Pull request merged**